ha/virsh/run_tests.sh

```diff
@@ -9,10 +9,10 @@ source /etc/profile.d/libvirt-uri.sh
 
 test_failed=0
 for file in $TESTS; do
-  AGENT=$(echo "$file" | grep -oP '(?<=/).+(?=\_)')
+  AGENT=$(echo "$file" | grep -oP '(?<=/).+(?=\_)' | tr _ -)
```

Comment:
Is this change relevant to this fix? More in general, why do we need it?

Reply:
`hostname` command is removing the `_` which makes the teste fail. If we use only `-` this is not a problem.